libraries/AP_CANManager/AP_CANManager.cpp

```diff
@@ -384,6 +386,62 @@ void AP_CANManager::log_retrieve(ExpandingString &str) const
     str.append(_log_buf, _log_pos);
 }
 
+#if HAL_GCS_ENABLED
+/*
+  handle MAV_CMD_CAN_FORWARD mavlink long command
+ */
+bool AP_CANManager::handle_can_forward(mavlink_channel_t chan, const mavlink_command_long_t &packet, const mavlink_message_t &msg)
```

Comment:
if needs access to class members, so can't be static. I don't think the extra class layer is worthwhile and just takes up flash for now - if/when we add this to AP_Periph we could work out how to make it available once we know how AP_Periph will interact with AP_CANManager

Reply:
static vars take memory even when we have CAN disabled. We shouldn't make things static unless we have a very good reason to do so.
When we come to enable this in AP_Periph we can revisit and work out how it will be structured